releases/release-6.0.0-dmr.md

```diff
@@ -286,49 +286,254 @@ TiDB v6.0.0 is a DMR, and its version is 6.0.0-DMR.
-| DM | [`loaders.<name>.import-mode`](/dm/task-configuration-file-full.md#task-configuration-file-template-advanced) | Newly added | The import mode during the full import phase. Since v6.0, DM uses TiDB Lightning's TiDB-backend mode to import data during the full import phase; the previous Loader component is no longer used. This is an internal replacement and has no obvious impact on daily operations.<br/>The default value is set to `sql`, which means using `tidb-backend` mode. In some rare cases, `tidb-backend` might not be fully compatible. You can fall back to Loader mode by configuring this parameter to `loader`. |
-| DM | [`loaders.<name>.on-duplicate`](/dm/task-configuration-file-full.md#task-configuration-file-template-advanced) | Newly added | Specifies the methods to resolve conflicts during the full import phase. The default value is `replace`, which means using the new data to replace the existing data. |
-| TiCDC | [`dial-timeout`](/ticdc/ticdc-sink-to-kafka.md#configure-sink-uri-for-kafka) | Newly added | The timeout in establishing a connection with the downstream Kafka. The default value is `10s`. |
-| TiCDC | [`read-timeout`](/ticdc/ticdc-sink-to-kafka.md#configure-sink-uri-for-kafka) | Newly added | The timeout in getting a response returned by the downstream Kafka. The default value is `10s`. |
-| TiCDC | [`write-timeout`](/ticdc/ticdc-sink-to-kafka.md#configure-sink-uri-for-kafka) | Newly added | The timeout in sending a request to the downstream Kafka. The default value is `10s`. |
+<table>
+<thead>
+  <tr>
+    <th>Configuration file</th>
+    <th>Configuration</th>
+    <th>Change type</th>
+    <th>Description</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>TiDB</td>
+    <td>
+      <code>stmt-summary.enable</code><br/>
+      <code>stmt-summary.enable-internal-query</code><br/>
+      <code>stmt-summary.history-size</code><br/>
+      <code>stmt-summary.max-sql-length</code><br/>
+      <code>stmt-summary.max-stmt-count</code><br/>
+      <code>stmt-summary.refresh-interval</code>
+    </td>
+    <td>Deleted</td>
+    <td>Configuration related to the <a href="https://docs.pingcap.com/tidb/dev/statement-summary-tables">statement summary tables</a>. All these configuration items are removed. You need to use SQL variables to control the statement summary tables.</td>
+  </tr>
+  <tr>
+    <td>TiDB</td>
+    <td><a href="https://docs.pingcap.com/tidb/dev/tidb-configuration-file#new_collations_enabled_on_first_bootstrap"><code>new_collations_enabled_on_first_bootstrap</code></a></td>
+    <td>Modified</td>
+    <td>Controls whether to enable support for the new collation. Since v6.0, the default value is changed from false to true. This configuration item only takes effect when the cluster is initialized for the first time. After the first bootstrap, you cannot enable or disable the new collation framework using this configuration item.</td>
```

Comment:
```suggestion
    <td>Controls whether to enable support for the new collation. Since v6.0, the default value is changed from <code>false</code> to <code>true</code>. This configuration item only takes effect when the cluster is initialized for the first time. After the first bootstrap, you cannot enable or disable the new collation framework using this configuration item.</td>
```